### ISSUE TITLE
TextBuffer.on is deprecated.

### ISSUE THREAD

**Issue by kaisellgren** [Nicolab/atom-local-history#21]
Use TextBuffer::onWillSave instead. A TextBuffer instance is no longer provided as a callback argument.
```
TextBuffer.on (C:\Users\kai\AppData\Local\atom\app-0.190.0\resources\app\node_modules\text-buffer\lib\text-buffer.js:1179:16)
<unknown> (C:\Users\kai\.atom\packages\local-history\lib\local-history.js:35:21)
```

**Comment by Nicolab** [comment]
ok I will change, thank you

**Comment by Nicolab** [comment]
Released to local-history@v3.0.0 (compatible Atom v0.192 and higher).

Thanks :)
